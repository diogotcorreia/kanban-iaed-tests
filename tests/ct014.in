a DEBUG 1
a
t 45 entretanto
t 60 sobretudo
t 75 praticamente
d DEBUG 1
q