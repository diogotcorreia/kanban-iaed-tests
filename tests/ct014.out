TO DO
IN PROGRESS
DONE
DEBUG 1
task 1
task 2
task 3
